client/components/thank-you/index.tsx

```diff
@@ -84,10 +99,12 @@ const ThankYouNextStep = ( props: ThankYouNextStepProps ) => {
 
 	return (
 		<React.Fragment key={ stepKey }>
-			<h3>{ stepTitle }</h3>
-			<div>
-				<p>{ stepDescription }</p>
-				<div>{ stepCta }</div>
+			<div className="thank-you__steps">
```

Comment:
Shouldn't this be `thank-you__step`? This is a single step.
```suggestion
			<div className="thank-you__step">
```